api/config/config_prod.json

```diff
@@ -152,7 +152,10 @@
   },
   "reporting": {
     "dataset": "reporting_prod",
-    "maxRowsPerInsert": 10000,
+    "maxRowsPerInsert": 500,
+    "batchSizeOverrides": {
+      "dataset_domain_value": 5000
```

Comment:
this table is known to be very slow (~ 7 minutes) at a batch size of 500.  Keeping the change from https://github.com/all-of-us/workbench/pull/9100